keda/README.md

```diff
@@ -63,8 +63,9 @@ their default values.
 | `crds.install`                               | Defines whether the KEDA CRDs have to be installed or not. | `true`                 |
 | `watchNamespace`                                           | Defines Kubernetes namespaces to watch to scale their workloads. Default watches all namespaces | `` |
 | `operator.name`                                            | Name of the KEDA operator | `keda-operator` |
-| `operator.replicaCount`                                      | Capability to configure the number of replicas for KEDA operator.<br /><br />While you can run more replicas of our operator, only one operator instance will be the leader and serving traffic.<br /><br />You can run multiple replicas, but they will not improve the performance of KEDA, it could only reduce downtime during a failover.| `1` |
+| `operator.replicaCount`                                      | Capability to configure the number of replicas for KEDA operator.<br /><br />While you can run more replicas of our operator, only one operator instance will be the leader and serving traffic.<br /><br />You can run multiple replicas, but they will not improve the performance of KEDA, it could only reduce downtime during a failover.<br /><br />Learn more in [our documentation](https://keda.sh/docs/latest/operate/cluster/#high-availability).| `1` |
 | `operator.affinity`                                        | Affinity for pod scheduling ([docs](https://kubernetes.io/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity/)) for KEDA operator. Takes precedence over the `affinity` field | `{}` |
+| `metricsServer.replicaCount`                                      | Capability to configure the number of replicas for KEDA metric server.<br /><br />While you can run more replicas of our metric server, only one instance will used and serve traffic.<br /><br />You can run multiple replicas, but they will not improve the performance of KEDA, it could only reduce downtime during a failover.<br /><br /> Learn more in [our documentation](https://keda.sh/docs/latest/operate/cluster/#high-availability).| `1` |
```

Comment:
This is not totally true, use all or not depends on an parameter but AKS, EKS and GKE AFAIK only query to one all the time.
IDK if it makes sense to clarify this or it's too in depth.
I could look for the parameter name because I think I have it anywhere if you want to clarify it

Reply:
Can you elaborate please? I don't get what you are trying to say?

Reply:
I don't get it as well. The description is quite accurate.

Reply:
I remember a test we did where all the requests were routed to the same metrics server, but you told me @zroubalik  that it's a configuration parameter that OpenShift allows to override and you can also modify on bare metal clusters

Reply:
I can't figure out when we talked about this, so nvm